azurerm/internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -408,3 +447,30 @@ func flattenAppConfigurationAccessKey(input appconfiguration.APIKey) []interface
 		},
 	}
 }
+
+func expandAppConfigurationIdentity(d *schema.ResourceData) *appconfiguration.ResourceIdentity {
+	identities := d.Get("identity").([]interface{})
+	if len(identities) == 0 {
+		return &appconfiguration.ResourceIdentity{}
+	}
+	identity := identities[0].(map[string]interface{})
+	identityType := appconfiguration.IdentityType(identity["type"].(string))
+	return &appconfiguration.ResourceIdentity{
+		Type: identityType,
+	}
+}
+func flattenAppConfigurationIdentity(identity *appconfiguration.ResourceIdentity) []interface{} {
+	if identity == nil {
```

Comment:
Here we also need to check if the identity's type is None, if so we should also return an empty identity block, in case a user is importing a app_configuration that is created with an explicit None identity.

Reply:
Hmm, I pushed a change here, but I'm not completely sure whether "return an empty identity block" means returning just an empty array of interfaces or something else, so please correct me if I'm wrong. Would appreciate some guidance here :)